3main

```diff
@@ -209,17 +202,17 @@ then
   echo ""
   echo "Restoring Xorg"
   echo ""
-  if [ -e /etc/X11/xorg.conf.default ]
+  if [[ -e ${NVOC}/xorg.conf.default ]]
   then
     echo "Restore default xorg.conf"
-    sudo cp /etc/X11/xorg.conf.default /etc/X11/xorg.conf
+    sudo cp ${NVOC}/xorg.conf.default /etc/X11/xorg.conf.default
   else
     echo "Downloading default xorg.conf"
     wget -N https://raw.githubusercontent.com/papampi/nvOC_by_fullzero_Community_Release/19-2.1/xorg.conf.default -O /tmp/xorg.conf.default
```

Comment:
We are checking ${NVOC}/xorg.conf.default existence, why not download it to same location if not exists?
     wget -N https://raw.githubusercontent.com/papampi/nvOC_by_fullzero_Community_Release/19-2.1/xorg.conf.default -O ${NVOC}/xorg.conf.default

Reply:
Because you don't know which is the current branch, that file should be inside the repo clone, if it has not been manually deleted. In such cases there is no sense in storing the version bundled with current 19-2.1 tree inside the ${NVOC} folder which maybe on a different branch. The wget is only a fallback. If we store the wget result somewhere then we should anyway redownload it any time since if it is buggy and you always use the previously downloaded buggy version you are stuck.